### DISCUSSION TITLE
Regarding the configuration of wing position

### DISCUSSION THREAD

**Discussion by chittychen** [JSBSim-Team/jsbsim#1070] [Q&A]
I checked the official JSBSim documentation and found that it is not necessary to configure specific wing position information.

JSBSim doesn't have the concept of wings and fuselage, so how does it calculate the forces and directions acting on the aircraft? In which module does this configuration need to be done?

**Comment by seanmcleod** [comment]
Have you taken a look at the JSBSim reference manual mentioned at - https://github.com/JSBSim-Team/jsbsim?tab=readme-ov-file#user-documentation

It explains how forces and moments are calculated in JSBSim.
